README.md

```diff
@@ -153,13 +211,14 @@ resource "google_service_account" "my_service_account" {
   account_id   = "my-new-service-account"
 }
 
-module "iam_binding" {
-  source = "terraform-google-modules/iam/google"
-  mode   = "authoritative"
+module "folders_iam_binding" {
```

Comment:
Please recheck your IDEA formatting settings and fix this one

Reply:
I think it will be enough find . -type f -name "*.tf" -exec terraform fmt  {} \;